server/consts_t.go

```diff
@@ -18,5 +18,6 @@ const (
 	testMode                = true
 	wattTimeFrequency       = 20 * time.Millisecond
 
-	ReportMigrationFrequency = 100 * time.Millisecond
+	ReportMigrationFrequency        = 100 * time.Millisecond
+	WattTimeWeekDataUpdateFrequency = 1000 * time.Millisecond
```

Comment:
Seeing this, we should probably disable watttime requests during testing right? The test instances don't have access to the watttime api.

Reply:
I will look around and see if there are other tests that are running watttime and disable during testing.